common/hsm_encryption.c

```diff
@@ -2,6 +2,8 @@
 #include <common/hsm_encryption.h>
 #include <sodium/utils.h>
 #include <termios.h>
+#include <unistd.h>
+#include <stdio.h>	
```

Comment:
Hidden whitespace at the end of line makes it fail make check-source:

```
common/hsm_encryption.c:6:#include <stdio.h>	
Extraneous whitespace found
make: *** [Makefile:455: check-whitespace/common/hsm_encryption.c] Error 1
```

Damn our pedantic source checks!

(I would normally just push a fix directly, but some people consider that rude :)

Reply:
thank you, fixed in 07269b0. It's fair, every character counts!